build/Common.prod.props

```diff
@@ -61,8 +61,8 @@
 
   <!--PublicApi Analyzer-->
   <ItemGroup>
-    <AdditionalFiles Include=".publicApi\$(TargetFramework)\PublicAPI.Shipped.txt" />
-    <AdditionalFiles Include=".publicApi\$(TargetFramework)\PublicAPI.Unshipped.txt" />
+    <AdditionalFiles Include=".publicApi\*\PublicAPI.Shipped.txt" />
```

Comment:
@eddynaka Is this cool? Seems to build fine and now all the folders show up in VS.

Reply:
sure, no problem! 👍